src/Uno.Extensions.Core.Generators/Compat/CompatibilityTypesGenerationTool.cs

```diff
@@ -402,7 +402,9 @@ private bool GetIsDisabled(string propertyName)
 		}
 		using var resourceReader = new StreamReader(resource);
 
+#pragma warning disable RS1035 // Using Environment.NewLine is fine
```

Comment:
Fine for now, but not yet clear if we should be using `Environment.NewLine` or not. There is a bit of discussion around that in https://github.com/dotnet/roslyn-analyzers/issues/6467